script/tool/test/repo_package_info_check_command_test.dart

```diff
@@ -489,7 +489,6 @@ release:
       expect(
         output,
         containsAll(<Matcher>[
-          contains('  Checking ci_config.yaml...'),
```

Comment:
Removed additional print to be more in sync with all the check in repo_package_info